docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 services:
   ego-api:
-    image: "overture/ego:2.9.0"
+    image: "overture/ego:3.1.0"
```

Comment:
I'm not sure why these are falling too much behind .. I matched them to the genomic playground didn't want to change alot of things in this small scope ticket